staging/src/k8s.io/apiserver/pkg/admission/metrics/metrics.go

```diff
@@ -149,11 +150,22 @@ func newAdmissionMetrics() *AdmissionMetrics {
 		},
 		[]string{"name", "type", "operation", "error_type", "rejection_code", "namespace"})
 
+	webhookRequest := metrics.NewCounterVec(
+		&metrics.CounterOpts{
+			Namespace:      namespace,
+			Subsystem:      subsystem,
+			Name:           "webhook_request_count",
```

Comment:
According to [prometheus best practices](https://prometheus.io/docs/practices/naming/), counters should be suffixed by `_total`.
```suggestion
			Name:           "webhook_request_total",
```

Reply:
Done.